ljbuild.make

```diff
@@ -0,0 +1,92 @@
+.SECONDEXPANSION:
```

Comment:
What's this doing?

Reply:
I needed it in the past to delay expansion of variable names but i don't see any $$ variable names left so it can probably go.

Reply:
fixed